src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -197,6 +201,19 @@ function validate_boolean_string_value(value) {
     return false;
 }
 
+function validate_supplemental_groups(value) {
+    if (value && typeof value === 'string') {
```

Comment:
need to add the same validation on account_server/accountspace_FS
CC: @shirady

Reply:
I'm not sure I understand - in which context do you want to add those validations?